cristin-import/src/test/resources/features/GeneralMappingRules.feature

```diff
@@ -69,6 +69,16 @@ Feature: Mappings that hold for all types of Cristin Results
     When the Cristin Result is converted to an NVA Resource
     Then the NVA Resource has a Creation Date equal to "2011-12-03T00:00:00Z"
 
+  Scenario: The NVA Resource Published Date is set to be the Cristin entry's creation date
+    Given that Cristin Result has created date equal to the local date "2011-12-03"
+    When the Cristin Result is converted to an NVA Resource
+    Then the NVA Resource has a Published Date equal to "2011-12-03T00:00:00Z"
+
+  Scenario: The NVA Resource Modified Date is set to be the Cristin entry's last modified date
```

Comment:
When the modified date is null the modified date should be equal to created date